### DISCUSSION TITLE
PDF dynamic paper size

### DISCUSSION THREAD

**Discussion by ArniSamano** [YetiForceCompany/YetiForceCRM#16040] [How to]
Come across question: how to generate pdf with dynamic length adjusted to content. The use case is to print receipts on continuouse paper roll (POS printer). Any one have idea how to implement such thing?

Arni

**Comment by s2b-git** [comment]
Something like that could be implemented for this case: http://150.109.21.51:81/pdf-pages-merge.aspx
They call it stitching 

Best
